lib/issue_badge_application_hooks.rb

```diff
@@ -1,19 +1,18 @@
 class IssueBadgeApplicationHooks < Redmine::Hook::ViewListener
   include ApplicationHelper
 
-  def view_layouts_base_html_head(context = {})
+  def view_layouts_base_html_head(_context = {})
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for view_layouts_base_html_head is too high. [18.92/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [12/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)